### ISSUE TITLE
PHPStan include gives cannot find  ArgumentsMapperPHPStanExtension service error

### ISSUE THREAD

**Issue by simPod** [CuyZ/Valinor#278]
I have included the phpstan config as stated in the docs
```neon
includes:
    - phpstan-baseline.neon
    - vendor/cuyz/valinor/qa/PHPStan/valinor-phpstan-configuration.php
```

but when I run phpstan, I receive

`Service (CuyZ\Valinor\QA\PHPStan\Extension\ArgumentsMapperPHPStanExtension::__construct()): Class 'CuyZ\Valinor\QA\PHPStan\Extension\ArgumentsMapperPHPStanExtension' not found.`

error. Is it just me? Used to work with 0.x.

**Comment by romm** [comment]
Hi @simPod this issue was solved in #268, this should be running in the upcoming release. In the meantime are you able to patch your app?

Thanks!

**Comment by simPod** [comment]
Olala, missed that one. Thanks.
